docroot/sites/all/modules/custom/ppgetstat/ppcmnt/ppcmnt.module

```diff
@@ -143,27 +144,24 @@ function _ppcmnt_parse_posts_list($page_content, $data) {
  * Parse post page for comments.
  */
 function _ppcmnt_parse_post_page($page_content, $data) {
-
   $user_data = json_decode($page_content);
-  $count = count($user_data->list);
+  $end = REQUEST_TIME - PPGETSTAT_DEFAULT_TIME_PERIOD_FOR_SCANNING;
 
-  $comments_counter_array = [];
-  $list = &$user_data->list;
+  $result = [];
 
-  $end = REQUEST_TIME - variable_get('ppgetstat_stats_period');
-  for ($i = ($count - 1); $i >= 0; $i--) {
-    if ($list[$i]->created < $end) {
+  foreach ($user_data->list as $i => $item) {
```

Comment:
Desc loop is done for scanning last users first. Is this construction do the same?